From fcaa289f6015595a638eb96b9f728eaaa7b13ab8 Mon Sep 17 00:00:00 2001
From: Michael Catanzaro <mcatanzaro@redhat.com>
Date: Tue, 23 Sep 2025 12:13:58 -0500
Subject: [PATCH] Fix build after 298234.133@webkitglib/2.50
 https://bugs.webkit.org/show_bug.cgi?id=299018

Unreviewed build fix for various architectures. Thanks to Yijia for
helping and to Justin Michaud for providing this fix.

Canonical link: https://commits.webkit.org/298234.154@webkitglib/2.50
---
 Source/JavaScriptCore/llint/LowLevelInterpreter32_64.asm | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Source/JavaScriptCore/llint/LowLevelInterpreter32_64.asm b/Source/JavaScriptCore/llint/LowLevelInterpreter32_64.asm
index 6d4036a55f96..f49782964262 100644
--- a/Source/JavaScriptCore/llint/LowLevelInterpreter32_64.asm
+++ b/Source/JavaScriptCore/llint/LowLevelInterpreter32_64.asm
@@ -3354,6 +3354,9 @@ op(loop_osr_entry_gate, macro ()
     crash() # Should never reach here.
 end)
 
+op(op_instanceof_return_location, macro ()
+    crash() # Should never reach here.
+end)
 
 llintOpWithMetadata(op_check_private_brand, OpCheckPrivateBrand, macro (size, get, dispatch, metadata, return)
     metadata(t5, t2)
-- 
2.51.0

